cfgov/core/management/commands/inactive_users.py

```diff
@@ -0,0 +1,63 @@
+from datetime import timedelta
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.core import mail
+from django.utils import timezone
+from django.utils.formats import date_format
+
+
+class Command(BaseCommand):
+    help = 'Find users who have been inactive for a given amount of time'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--period',
+            type=int,
+            default=90,
+            help='The number of days that defines inactivity'
+        )
+        parser.add_argument(
+            '--emails',
+            nargs='+',
+            default=[],
+            help='Email output to a list of addresses'
+        )
+
+    def handle(self, *args, **options):
+        period = options['period']
+        emails = options['emails']
+
+        last_possible_login = timezone.now() - timedelta(days=period)
+
+        User = get_user_model()
+        inactive_users = User.objects.filter(
+            is_active=True, last_login__lt=last_possible_login)
+
+        if len(inactive_users) == 0:
+            return
+
+        if len(emails) > 0:
+            self.stdout.write('Sending inactive user list to '
+                              '{}\n'.format(','.join(emails)))
+            self.send_email(emails, period, inactive_users)
+        else:
```

Comment:
Minor thing, but it might be nice to always have the list of users displayed even if the email message option is chosen. This'll make it easier to review e.g. job logs in Jenkins where emails were sent out.